FutureNorthants/northants-design-system
# Add bin finder slice and bin collections component

There are two components:
- The BinFinder slice will send the requests to the API to find the addresses UPRN and then send a request to the new API to get the collection details. The slice will need to be inserted into the page using Drupal. 
- The BinCollection component will display the collection data. 

[Bin collections component design](https://deploy-preview-409--northants-design-system.netlify.app/?path=/story/library-components-bincollection--example-bin-collection&globals=theme:west_theme)

[Bin Finder slice design](https://deploy-preview-409--northants-design-system.netlify.app/?path=/story/library-slices-binfinder--example-bin-finder&globals=theme:west_theme)

[Bin collection day example page](https://deploy-preview-409--northants-design-system.netlify.app/?path=/story/page-examples-bin-collection-day--bin-finder-example&globals=theme:west_theme)

## Testing
- Checkout this branch and run `npm install` to install dependencies
- Run `npm run dev` and view the Bin collections component and Bin finder slice
- View the Bin collection example page and test searching for various postcodes to ensure the bin collection data matches the expected response. 
- Manually run tests with `npm run test`
